python/rikai/spark/sql/codegen/mlflow_logger.py

```diff
@@ -128,6 +129,9 @@ def log_model(
                     "will not be populated to MLflow"
                 )
         mlflow.set_tags(tags)
+        model_registry_client = MlflowClient()._get_registry_client()
+        for key, value in tags.items():
+            model_registry_client.set_registered_model_tag(registered_model_name, key, value)
```

Comment:
Question here, should the tag attach to a register model or a register model version?

Can user change some of these flags from different versions

Reply:
this is the issue I was complaining to you about. When you call mlflow log_model, it creates a new version but doesn't return the version id to you. My thought is that we probably want to log properties to both the model and the version. When you create a new version, the latest version's properties should override the ones on the registered model. The intended workflow is that most of the time, users will search registered models and then just use the latest. And if there's some debugging workflow, they can click through historical versions and try to find any changes.